mods/ra/sequences/structures.yaml

```diff
@@ -143,8 +138,6 @@ apwr:
 	make: apwrmake
 		Start: 0
 		Length: *
-	dead: apwrdead
-		Start: 0
```

Comment:
:cry: It is a bit sad to see the features I added over the months removed one by one, but it seems I have been outvoted at https://github.com/OpenRA/OpenRA/issues/3522 in keeping an incomplete set so I guess that is okay then.